core/tests/firebase_admin_sdk_stub_test.py

```
@@ -0,0 +1,206 @@
+    def test_get_auth_claims_returns_none_when_session_inactive(self) -> None:
+        os.environ['USER_ID'] = 'user_123'
+        os.environ['USER_EMAIL'] = 'test@example.com'
+        setattr(self.stub, '_is_session_active', False)
+        claims = self.stub.get_auth_claims_from_request(self.mock_request)
+        self.assertIsNone(claims)
+
+    def test_get_auth_claims_returns_claims_when_session_active(self) -> None:
+        os.environ['USER_ID'] = 'user_123'
+        os.environ['USER_EMAIL'] = 'test@example.com'
+        os.environ['USER_IS_ADMIN'] = '0'
+        setattr(self.stub, '_is_session_active', True)
```

Ruff (0.15.14)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)